packages/core/EventManager.ts

```diff
@@ -545,6 +561,14 @@ export default class EventManager {
           );
           // It might not be the first connected calendar as it seems that the order is not guaranteed to be ascending of credentialId.
           const firstCalendarCredential = destinationCalendarCredentials[0];
```

Comment:
This check is assuming that there is a second calendar credential of the same type as the original destination calendar.